### ISSUE TITLE
qase.step exception if qs_enabled = False in pytest.ini

### ISSUE THREAD

**Issue by Nickolasll** [qase-tms/qase-python#51]
If I disable qase plugin for debug I have this exception

```
core/test_runner.py:162: in run_test_case
    with qase.step(self.command_number):
/usr/lib/python3.8/contextlib.py:113: in __enter__
    return next(self.gen)
/usr/local/lib/python3.8/dist-packages/qaseio/pytest/__init__.py:120: in step
    raise e
/usr/local/lib/python3.8/dist-packages/qaseio/pytest/__init__.py:113: in step
    plugin = QasePytestPluginSingleton.get_instance()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    @staticmethod
    def get_instance() -> QasePytestPlugin:
        """ Static access method"""
        if QasePytestPluginSingleton._instance is None:
>           raise Exception("Init plugin first")
E           Exception: Init plugin first

/usr/local/lib/python3.8/dist-packages/qaseio/pytest/plugin.py:478: Exception
```

Plugin was not init, because qs_enabled = False, but It was ignored in qase.step

**Comment by EugenSusurrus** [comment]
Hi @Nickolasll ,

this was actually fixed in this PR https://github.com/qase-tms/qase-python/pull/41

But for some reason the qase team did not deploy the fix to production. It is on the master branch, but when you `pip install pytest-qase` you still get the previously deployed version of the plugin `3.0.0`.

Maybe someone from @sklmx @n3r @zeburek @mgolovko can have a look.

Actually at our company we rely on uploading results to qase in real time, and this fix which was not yet deployed. Therefore I'a also like to ask if it were possible to create a `hotfix` branch for the `qase-pytest == 2.3.0` version to also merge the changes from https://github.com/qase-tms/qase-python/pull/41 into?

So the plugin which works with the old qaseio client is also fixed? That would be awesome actually!

**Comment by sklmx** [comment]
Hi! The last changes were published. 
@EugenSusurrus try it out and thanks for reminding :)

**Comment by sharovatov** [comment]
Hey @EugenSusurrus , we've released 3.0.1 version :)

https://github.com/qase-tms/qase-python/releases/tag/qase-pytest-3.0.1

**Comment by Nickolasll** [comment]
It works great on 3.0.1 version, thanks!

**Comment by EugenSusurrus** [comment]
@sklmx Thanks a ton!

What about ?

> Therefore I'a also like to ask if it were possible to create a hotfix branch for the qase-pytest == 2.3.0 version to also merge the changes from https://github.com/qase-tms/qase-python/pull/41 into?

Or should I create a new PR?

**Comment by EugenSusurrus** [comment]
Because we can't yet update to the 3.0.1 plugin since it only uploads results in bulk as I understood and we want to upload per test case as the test runs executes, which is only supported in 2.3.0.

So should I create a hotfix for 2.3.0?

**Comment by sklmx** [comment]
@EugenSusurrus that'd be the fastest way to apply a fix. Although I'm still wondering why you can't use the bulk method. Describe your motivation in the new issue/pull request, please.

**Comment by EugenSusurrus** [comment]
@sklmx This desired behaviour and rationale is basically described in this ticket. Though it refers to the new pytest plugin (>3.0.0) which uses the new qaseio client which only uploads in bulk.

https://github.com/qase-tms/qase-python/issues/50

The old client meets better our requirements because it uploads the results after each test case.

Probably it would be nice to have an option in which you could upload "real-time" or in bulk. In this case, longer running test cases could upload results as they complete, and for the test runs with faster test cases, uploading in bulk would maybe make sense if there is an qase api rate limit for example.

**Comment by EugenSusurrus** [comment]
@sklmx I've got some free time on my hands. Basically I would like to apply changes from #41 to `qase-pytest == 2.3.2`.

Since I can't find a release branch tagged with this version, how should I proceed in order to create a fix version 2.3.3 which is going to be released with the aforementioned changes in #41? Basically the same `__init__.py` file has to be modified in  the same way (more or less), for a 2.3.3 to work as intended when running test cases with steps without the `--qase` flag.

Can someone create a branch from https://github.com/qase-tms/qase-python/tree/qase-pytest-v2.3.2/qase-pytest to which I can create my PR? What would the workflow be in that case?

Many thanks!
